jobspec.yaml

```diff
@@ -1,30 +1,30 @@
 spec:
   declarative: true
-  git_url: https://gitlab.cee.redhat.com/coreos/rhcos-upshift.git
-  git_ref: master
+  git_url: https://github.com/openshift/ocp-build-data
+  git_ref: rhcos-4.1
 cosa:
   override: false
   image: coreos-assembler:latest
 job:
-  build_name: ben-ootpa
-  brew_profile: brew-stage
+  build_name: rhcos-4.1
```

Comment:
note this is different from the existing build which is `rhcos-ootpa`; that's good because we'd like to try this out before wiping out the other one.